service/operator/xtradb_operator.go

```diff
@@ -44,10 +44,22 @@ func (x XtraDBOperatorService) InstallXtraDBOperator(ctx context.Context, req *c
 	}
 	defer client.Cleanup() //nolint:errcheck
 
-	err = client.InstallOperator(ctx, req.Version, x.manifestsURLTemplate)
+	operators, err := client.CheckOperators(ctx)
 	if err != nil {
 		return nil, err
 	}
 
+	err = client.ApplyOperator(ctx, req.Version, x.manifestsURLTemplate)
+	if err != nil {
+		return nil, err
+	}
+
+	if operators.XtradbOperatorVersion != "" {
```

Comment:
Should we rely on this?
There could be a situation when no operator is installed but there are clusters deployed. But it's very corner case situation. Most likely the cluster won't be working properly long term anyway.

Reply:
i am not sure about that but I think it would be a good idea to document that concern/edge case in a comment (here and in the psmdb code) at the very least

Reply:
Note added.